packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -22,273 +22,330 @@ related: [
-* Helper text should be specific to the input and not the label.
-* If there is specific information about the label it should be with the label. 
-* If needed, helper text can wrap to two lines, but it should never extend past the width of the input field.
-* Keep helper text concise and avoid exceeding the length of the input.
+Helper text is brief, permanent text below a form field that helps a user provide the right information specific to that input, like entering a unique project name or data point. Helper text may wrap to two lines, but it should never extend past the width of its corresponding input field.
 
+Align helper text with the specific input for which it offers assistance. If your helper text contains information about the label rather than the input field, place it with the label instead.
 
 #### Popovers
-Popovers are content boxes that are used for fields that might require additional background or explanation. You can also use a popover to link to external help pages or other related information. In forms, popovers are indicated by an unfilled question mark circle that a user can click on to reveal the information. They should be placed to the right of a form label. 
 
- <img src="./img/popover.png" alt="Example of popover content box used in a form field" width="411"/>
+Use a [popover](/components/popover/design-guidelines) for fields that might require additional background or explanation. Popovers may also link to external help pages or other related documentation. 
+
+In a form, indicate a popovers with an unfilled question mark circle that reveals information when clicked. Place popovers to the right of their corresponding form label.
 
-**Usage**
-* You may use popovers for content that requires formatting like numbered or bulleted lists.
-* Use popovers to guide users as to where they can find the information needed.
-* Use popovers for content that requires you to include buttons, links, or images. 
-* **Do not** hide critical information inside a popover that users would need in order to complete their task. 
+ <img src="./img/popover.png" alt="Example of a popover used in a form field to provide additional context for where a user can find the required information" width="411"/>
 
-For more information on user help, see our [embedded assistance guidelines](/components/popover/design-guidelines).
+ Use popovers to:
 
-### Use progressive disclosure
-Use progressive disclosure to hide and show fields based on the users’ selection. This approach keeps the workflow short and allows the user to focus on only the fields relevant to them. It also avoids using color to indicate whether or not a field is available, making it suitable for visually impaired users.
+* Provide content that requires formatting like numbered or bulleted lists, or that requires you to include additional elements like buttons, links, or images.
+* Guide users to where they can find essential information needed to complete a given form field.
```

Comment:
This is small but could you flip the order of these bullets? the second one is more pointed to its actual purpose where as the first one is more "additional" if that makes sense!